tools/workspace-plugin/src/plugins/clean-plugin.ts

```diff
@@ -63,3 +61,15 @@ function createNodesInternal(
     },
   };
 }
+
+export function buildCleanTarget(
```

Comment:
per plugin builders are exported so we can re-use the logic within workspace-plugin